### DISCUSSION TITLE
이번 챕터 5, 6장은 5월 19일 목요일에 올릴 예정입니다.

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#217] [General]
제가 지난 주에 자전거 사고로 한바퀴 구른 이후에 오른쪽 옆구리 쪽에 통증을 달고 살고 있습니다 ㅜㅜ
그리고 그 이후에 따로 온라인 멘토링 제외하고 뭔가 책 읽고 정리하거나 공부한 게 거의 없군요.
5월 18일 수요일에 연차를 내고 병원을 가는데 그 때 좀 읽고 정리해서 목요일 까지는 올려 볼 예정입니다.
가뜩이나 코로나 걸려서 체력도 저하됐는데 사고까지 나서 이래저래 몸상태가 안좋군요.

**Comment by Joseph-Cha** [comment]
아고.. 많이 다치신건 아닌지요?ㅠㅠ
저도 사실 다음주까지 사내 오픈 전 풀 야근 각이라 주중 참석이 어려웠었어요..
기간을 한 주 연기하는 건 어떨까요?

**Reply by jongfeel** [reply]
별다른 의견 없으면 동훈님 의견대로 1주일 연기하겠습니다.
그렇다고 하더라도 책 읽는 기간이 늘어나는 건 아니므로
책은 쭉 기존대로 읽는 걸 진행하고
6월 3일 실용주의 프로그래머 7장 모임도 그대로 진행할 예정입니다.
